diff --git a/jest/preprocessor.js b/jest/preprocessor.js
index f5e34763d840193e243a974e448b619f8f635095..cc6e05ab4c9c25a3d257379a1bf19c21fb9ef5ca 100644
--- a/jest/preprocessor.js
+++ b/jest/preprocessor.js
@@ -38,7 +38,7 @@ module.exports = {
         sourceType: 'script',
         ...nodeOptions,
         ast: false,
-      }).code;
+      });
     }
 
     const {ast} = transformer.transform({
@@ -66,8 +66,6 @@ module.exports = {
         [require('@babel/plugin-transform-flow-strip-types')],
         [
           require('@babel/plugin-proposal-class-properties'),
-          // use `this.foo = bar` instead of `this.defineProperty('foo', ...)`
-          {loose: true},
         ],
         [require('@babel/plugin-transform-computed-properties')],
         [require('@babel/plugin-transform-destructuring')],
@@ -112,7 +110,7 @@ module.exports = {
         sourceMaps: true,
       },
       src,
-    ).code;
+    );
   },
 
   getCacheKey: (createCacheKeyFunction([
