diff --git a/jest/createCacheKeyFunction.js b/jest/createCacheKeyFunction.js
index 890918962daebefedee59a48cd02906447f9de5b..2e73413d30a9964891608eb9ec02568b981b5ba0 100644
--- a/jest/createCacheKeyFunction.js
+++ b/jest/createCacheKeyFunction.js
@@ -32,6 +32,8 @@ function getGlobalCacheKey(files, values) {
 
 function getCacheKeyFunction(globalCacheKey) {
   return (src, file, configString, options) => {
+    // Jest 27 passes a single options bag which contains `configString` rather than as a separate argument
+    options = options || configString;
     const {instrument, config} = options;
     const rootDir = config && config.rootDir;
 
