diff --git a/jest/preprocessor.js b/jest/preprocessor.js
index 5920c0a6f23c056f27366fabf32dd13c6f86465b..2658e52f9127ac58849e7f830f6342d8b683672c 100644
--- a/jest/preprocessor.js
+++ b/jest/preprocessor.js
@@ -64,8 +64,6 @@ module.exports = {
         [require('@babel/plugin-transform-flow-strip-types')],
         [
           require('@babel/plugin-proposal-class-properties'),
-          // use `this.foo = bar` instead of `this.defineProperty('foo', ...)`
-          {loose: true},
         ],
         [require('@babel/plugin-transform-computed-properties')],
         [require('@babel/plugin-transform-destructuring')],
