diff --git a/docusaurus-plugin-sass.js b/docusaurus-plugin-sass.js
index 62180ebc48e43a21ad7b28de7e178c757941c69d..a4047541b2e84e19322276296cc161cdb1b29205 100644
--- a/docusaurus-plugin-sass.js
+++ b/docusaurus-plugin-sass.js
@@ -16,10 +16,10 @@ module.exports = function(_, {id, ...options}) {
                     localIdentName: isProd
                       ? `[local]_[hash:base64:4]`
                       : `[local]_[path]`,
+                    exportOnlyLocals: isServer,
                   },
                   importLoaders: 1,
                   sourceMap: !isProd,
-                  onlyLocals: isServer,
                 }), {
                   loader: 'sass-loader',
                   options: options || {}
